src/ui/left-panel/index.tsx

```diff
@@ -23,13 +23,27 @@ const openPanel = () => {
     }
 };
 
+const toSizeString = (value: number) => {
```

Comment:
could do with something like this https://gist.github.com/lanqy/5193417

Reply:
ok, switch to that version.